mongo/integration/sessions_prose_test.go

```diff
@@ -0,0 +1,27 @@
+package integration
+
+import (
+	"errors"
+	"testing"
+
+	"go.mongodb.org/mongo-driver/internal/testutil/assert"
+	"go.mongodb.org/mongo-driver/mongo/integration/mtest"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func TestSessionsProse(t *testing.T) {
+	mtOpts := mtest.NewOptions().
+		MinServerVersion("5.0").
```

Comment:
Minor: Do we need a wire version or topology check? The error occurs when constructing a session. That should not require connecting to a server.

Reply:
Ah fair point. Removed version and topology check and kept `CreateClient(false)`.